pkg/seccomp/seccomp.json

```diff
@@ -152,6 +152,7 @@
 				"fadvise64",
 				"fadvise64_64",
 				"fallocate",
+				"fanotify_init",
```

Comment:
This places no restrictions on the arguments. Will this allow a container running as root to make the kernel go OOM by using FAN_UNLIMITED_MARKS?

In other words do the restrictions described here need to be reflected in the policy?

```
The limitations imposed on an event listener created by a user without the CAP_SYS_ADMIN capability are as follows:
```

https://www.man7.org/linux/man-pages/man2/fanotify_init.2.html

Reply:
Ok. I assume then there's no underlying reason it was required to be in the list in the first place. Basically just duplication of the checks done by the kernel any way.